src/Scientist/Internals/GCHelper.cs

```diff
@@ -0,0 +1,132 @@
+﻿namespace GitHub.Internals
+{
+    using System;
+    using System.Reflection;
+    using System.Runtime.InteropServices;
+
+    internal static class GCHelper
+    {
+        private const string DebugConfigurationName = "DEBUG";
+        internal const string ReleaseConfigurationName = "RELEASE";
+        internal const string Unknown = "?";
+
+        public static bool IsMono { get; } = Type.GetType("Mono.Runtime") != null;
+
+        public static bool IsFullFramework =>
+#if (NET451)
+            true;
+#else
+            System.Runtime.InteropServices.RuntimeInformation.FrameworkDescription.StartsWith(".NET Framework", StringComparison.OrdinalIgnoreCase);
```

Comment:
Just checking, these aren't strings that can vary if language packs are installed on the local machine?

Reply:
Also Hey Martin, I thought that looked like your name in the commits but thought it was a coincidence.